endpoints.json

```diff
@@ -103,6 +104,21 @@
       "name": "do_nothing",
       "avatar_url": "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png"
     }]
+  },
+  "GET /api/articles/?topic": {
+    "description": "serves an array of all articles filtered by topic",
+    "queries": [],
+    "exampleResponse": [
+      {
+        "title": "Seafood substitutions are increasing",
+        "topic": "cooking",
+        "author": "weegembump",
+        "body": "Text from the article..",
+        "created_at": "2018-05-30T15:59:13.341Z",
+        "votes": 0,
+        "comment_count": 6
+      }
+    ]
```

Comment:
You dont need to include this as your previous entry for /api/articles already has this with the queries array filled with the correct info including topic.